pkg/serverless/logs/logs.go

```diff
@@ -265,6 +266,10 @@ func (c *LambdaLogsCollector) ServeHTTP(w http.ResponseWriter, r *http.Request)
 }
 
 func processLogMessages(c *LambdaLogsCollector, messages []logMessage) {
+	// sort messages by time (all from the same time zone) in ascending order.
+	sort.Slice(messages, func(i, j int) bool {
+		return messages[i].time.String() < messages[j].time.String()
```

Comment:
```suggestion
		return messages[i].time.Before(messages[j].time)
```

Go's got some pretty great [handling of time objects](https://pkg.go.dev/time#Time.Before). (Please check that I've done this correctly, there's also a `Time.After` method)